build-scripts/build_llvm.py

```diff
@@ -7,6 +7,7 @@
 import argparse
 import os
 import pathlib
+import requests
```

Comment:
By adding this we probably also need to add `requirements.txt` as this is not a built in module.

Reply:
@loganek shall we create a global requirements.txt under */repo* or just create a local requirements.txt for *build_llvm.py* only.

Reply:
done, but i've added it locally under build-scripts

Reply:
I think we can keep the file local; looks like that's what we do for `wasi-nn` tests and `language-bindings` for python as well.

Reply:
👍 , kept it local for the same reason